addons/l10n_fr_pos_cert/models/res_company.py

```diff
@@ -11,7 +11,7 @@
 def ctx_tz(record, field):
     res_lang = None
     ctx = record._context
-    tz_name = pytz.timezone(ctx.get('tz') or record.env.user.tz)
+    tz_name = pytz.timezone(ctx.get('tz') or record.env.user.tz or 'UTC')
```

Comment:
@pimodoo I remember you told me one module like this had been hashed and cannot be changed anymore. Which one is it?

Reply:
@william-andre it is the belgian one that cannot be changed in stable